lms/djangoapps/verify_student/views.py

```diff
@@ -946,6 +947,27 @@ def _send_email(user_id, subject, message):
     user.email_user(subject, message, from_address)
 
 
+def _set_requirement_status(attempt, namespace, status, reason=None):
+    """Set requirement status and the reason of status for given user in
+    attempt.
+    """
+    checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+    if checkpoints:
```

Comment:
You can use [`exists()`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.exists) to more cleanly perform the check you're doing here. Then, assuming the first checkpoint is always the one you need, use `first()` to only load the first checkpoint, instead of all of them.